package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "graphql-shield",
+  "name": "grabcad-graphql-shield",
```

Comment:
We should rename the repository `grabcad-graphql-shield`

Reply:
i think leaving it like it is makes it more obvious that its a fork and not something private.  anyway i expect we will drop this fork when new version of shield comes out with better cache support, there are already some open issues with PRs around this same thing.